components/esp_lcd/src/esp_lcd_panel_rgb.c

```diff
@@ -282,7 +284,8 @@ esp_err_t esp_lcd_new_rgb_panel(const esp_lcd_rgb_panel_config_t *rgb_panel_conf
     periph_module_reset(lcd_periph_signals.panels[panel_id].module);
 
     // allocate frame buffers + bounce buffers
-    ESP_GOTO_ON_ERROR(lcd_rgb_panel_alloc_frame_buffers(rgb_panel_config, rgb_panel), err, TAG, "alloc frame buffers failed");
+    ret = lcd_rgb_panel_alloc_frame_buffers(rgb_panel_config, rgb_panel);
+    ESP_GOTO_ON_ERROR(ret, err, TAG, "alloc frame buffers failed");
```

Comment:
I assume this was changed for debugging. does it need to remain changed?

Reply:
I thought this is for correctness. I didn't think ESP_GOTO_ON_ERROR set `ret` based on the code after this. Turns out it does. Will remove.